### PR TITLE
fix: amend hospital clerk scope

### PR DESCRIPTION
**Pull request by Nil20**
## Description

Clearly describe what has been changed. Include relevant context or background.
Explain how the issue was fixed (if applicable) and the root cause.

Link this pull request to the GitHub issue (and optionally name the branch `ocrvs-<issue #>`)

## Checklist

- [ ] I have linked the correct Github issue under "Development"
- [ ] I have tested the changes locally, and written appropriate tests
- [ ] I have tested beyond the happy path (e.g. edge cases, failure paths)
- [ ] I have updated the changelog with this change (if applicable)
- [ ] I have updated the GitHub issue status accordingly

### CODE REVIEW

File: src/data-seeding/roles/roles.ts [hunk 1]
```diff
@@ -255,7 +255,8 @@ export const roles: Role[] = [
       'record.create[event=birth|death|tennis-club-membership]',
       'record.read[event=birth|death|tennis-club-membership]',
       'record.declare[event=birth|death|tennis-club-membership]',
-      'record.notify[event=birth|death|tennis-club-membership]'
+      'record.notify[event=birth|death|tennis-club-membership]',
+      'record.registered.print-certified-copies[event=tennis-club-membership,templates=v2.tennis-club-membership-certificate-alpha]'
```
Comment (Nil20) [inline src/data-seeding/roles/roles.ts:259]: added for `k.bwalya`

**Review by tahmidrahman-dsi — APPROVED** [review]

**Review by Zangetsu101 — APPROVED** [review]
